website/docs/en/plugins/rspack/subresource-integrity-plugin.mdx

```diff
@@ -0,0 +1,184 @@
+import { ApiMeta } from '@components/ApiMeta.tsx';
+
+# SubresourceIntegrityPlugin
+
+<ApiMeta specific={['Rspack']} addedVersion="1.2.4" />
+
+The `rspack.experiments.SubresourceIntegrityPlugin` is a plugin for enabling Subresource Integrity in Rspack.
+
+> [Subresource Integrity](http://www.w3.org/TR/SRI/) (SRI) is a security
```

Comment:
How about copying the "What is SRI" section from the Rsbuild documentation:

https://github.com/web-infra-dev/rsbuild/blob/71ea28b23f37c25303017870740bbb495b805683/website/docs/en/config/security/sri.mdx?plain=1#L16